tests/helpers/helpHelper-test.js

```diff
@@ -0,0 +1,7 @@
+// No tests needed ATM; all API Requests
+// parseMarkdown, unescapeInlineHtml?
```

Comment:
I think these two functions should be tested - they are different from most other helpers and might require a different format.

Reply:
@ebdabbs ,

I think mocking the data for these functions is a big effort. Not sure what's going on here and seems like we're using a third party library `markdown` to do some dirty work here.

These are the functions I mentioned that I tried to test but mocking the data for the parameters proved troublesome.

Reply:
https://federal-spending-transparency.atlassian.net/browse/DEV-2918